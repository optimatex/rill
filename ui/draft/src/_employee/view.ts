export {
  React,
  useState,
  useCallback,
  css,
  styled,
  createGlobalStyle,
} from '@rill/view';
