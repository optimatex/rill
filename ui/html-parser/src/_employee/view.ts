export { React, RC } from '@rill/view';
