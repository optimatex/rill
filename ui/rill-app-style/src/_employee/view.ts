export { createGlobalStyle, css } from '@rill/view';
