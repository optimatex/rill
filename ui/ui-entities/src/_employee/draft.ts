export { TextEditor } from '@rill/draft';
