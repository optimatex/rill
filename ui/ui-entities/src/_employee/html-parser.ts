export { htmlParser } from '@rill/html-parser';
