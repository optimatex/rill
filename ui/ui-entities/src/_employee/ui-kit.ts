export {
  // Layout
  Row,
  ListContainer,
  // Card
  Card,
  Post,
  Panel,
  // Menu
  MenuItemLink,
  // Typography
  Paragraph,
  Name,
  // Image
  Image,
  ImageCropper,
  ImagesLayout,
  // Video
  YoutubeVideos,
  // Content
  RichContentContainer,
} from '@rill/ui-kit';
