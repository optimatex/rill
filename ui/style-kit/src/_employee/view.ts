import { css as originalCss } from '@rill/view';

export const css = originalCss;
