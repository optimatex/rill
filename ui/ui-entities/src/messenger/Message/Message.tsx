import { React } from '_employee/view';
import { Row, Image, ImageCropper } from '_employee/ui-kit';
import { IMessage } from '_employee/rill-meta';

import MessageContent from '../MessageContent';

const Message = ({ text, author }: IMessage) => (
  <Row gap={12}>
    <ImageCropper>
      <Image src={author.avatarUrl} />
    </ImageCropper>

    <MessageContent html={text} />
  </Row>
);

export default Message;
