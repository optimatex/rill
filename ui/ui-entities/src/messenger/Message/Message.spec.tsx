import { React } from '_employee/view';
import { render, mockMessage_1 } from 'test';
import Message from '.';

describe('Message', () => {
  it('should render without errors', () => {
    const { container } = render(<Message {...mockMessage_1} />);

    expect(container).toMatchSnapshot();
  });
});
