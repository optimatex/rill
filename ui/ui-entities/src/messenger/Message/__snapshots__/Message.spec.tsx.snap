// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render without errors 1`] = `
.c2 {
  object-fit: cover;
  max-width: 100%;
  display: block;
}

.c1 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  height: 30px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0 auto;
  width: 30px;
  overflow: hidden;
  border-radius: 50px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  box-sizing: border-box;
  padding: 5px 8px;
}

.c0 > * {
  margin-right: 16px;
}

.c0 > *:last-child {
  margin-right: 0;
}

.c0 > * {
  margin-right: 12px;
}

.c4 {
  font-size: 8px;
  font-family: roboto;
  font-weight: bolder;
}

.c3 p {
  padding: 0;
  margin: 0;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <img
        class="c2"
        src="https://i.pravatar.cc/60"
      />
    </div>
    <div
      class="c3"
    >
      <p
        class="c4"
      >
        I've been a QA engineer with comp.sci background working in many places building quality tools for over 11 years. Last year during the pandemic most of my company was fired, including myself. So, I decided to apply for a web developer role at Napster. This video was literally all I used to do the technical exam. I am now finishing my probation period! Thanks, mate!
      </p>
    </div>
  </div>
</div>
`;
